src/frontend/cunningham.cjs

```diff
@@ -18,53 +18,55 @@
 */
 
 module.exports = {
-  theme: {
-    font: {
-      families: { base: 'Hind', accent: 'Montserrat' },
+  themes: {
+    default: {
```

Comment:
You must nest colors, font into `theme` ( see here https://openfun.github.io/cunningham/?path=/docs/migrating-from-v1-to-v2--docs#configuration-file ) 

`themes.detault.theme.colors`